### DISCUSSION TITLE
Being able to Right-click to scroll in song select

### DISCUSSION THREAD

**Discussion by Metalproz** [ppy/osu#24757] [Ideas] [answered]
In stable you can right click in an empty field to quickly go up/down and scroll the whole song select. Would def be very useful to quickly go down the list (esp. when u have difficulty filter enabled, you could quickly switch from 4 stars to 7 stars.

**Comment by soopax** [comment, marked as answer]
Try searching for "Right mouse drag to absolute scroll" in settings and turn it on.
